src/installLogsPrinter.d.ts

```diff
@@ -2,21 +2,26 @@
 
 declare function installLogsPrinter(on: Cypress.PluginEvents, options?: installLogsPrinter.PluginOptions): void;
 declare namespace installLogsPrinter {
+  enum Severity {
+    NONE = '',
+    ERROR = 'error',
+    WARNING = 'warning'
+  }
 
-  type Severity = '' | 'error' | 'warning';
-
-  type LogType = 'cons:log' |
-    'cons:info' |
-    'cons:warn' |
-    'cons:error' |
-    'cons:debug' |
-    'cy:log' |
-    'cy:xhr' |
-    'cy:fetch' |
-    'cy:request' |
-    'cy:intercept' |
-    'cy:command' |
-    'ctr:info';
+  enum LogType {
```

Comment:
Are you sure this works correctly? As far as I know this would also require exporting an object like this. And we aren't at this point, however you could implement that part as well.

Reply:
Yeah I see what you mean, we would need to export a const. It's really not needed, I'll cancel this PR